en/flight_stack/controller_diagrams.md

```diff
@@ -46,20 +46,34 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
 * When tuning this controller, the only parameter of concern is the P gain.
 * The rate command is saturated.
 
+### Multicopter Acceleration to Thrust and Attitude setpoint conversion
+* The acceleration setpoints generated by the velocity controller will be converted to thrust and attitude setpoints
+* Implementation details can be found in `ControlMath.cpp`
+* Converted acceleration setpoints will be saturated and priotized in vertical and horizontal thrust. 
+* Thrust saturation is done after computing the corresponding thrust:
+ 1. compute required vertical thrust (`thrust_z`)
+ 2. saturate `thrust_z` with `MPC_THR_MAX`
+ 3. saturate `thrust_xy` with `(MPC_THR_MAX^2 - thrust_z^2)^0.5`
+
 ### Multicopter Velocity Controller
 
-![MC Velocity Control Diagram](../../assets/diagrams/mc_velocity_diagram.jpg)
+![MC Velocity Control Diagram](../../assets/diagrams/mc_velocity_diagram.png)
```

Comment:
FWIW Personally I prefer consistency in diagram file type, and generally PNG results in cleaner images for similar file sizes. So I like this. Note though, there is no rule on this.